kubejs/server_scripts/gregtech/recipes.recycling.js

```diff
@@ -0,0 +1,44 @@
+// priority: 0
+
+function registerGTCEURecyclingRecipes(event) {
```

Comment:
the way I did the other recycling functions, is that the "main" recipe registering function (in recipes.js) would call all the "extra" functions (like the recycling ones) at the very top - stops main_server_script.js from getting so crowded